erc20-connector/dist/BridgeTokenFactory.full.sol

```diff
@@ -529,7 +529,7 @@ library Borsh {
         }
     }
 
-    function decodeBytes20(Data memory data) internal pure returns(bytes20 value) {
+    function decodeBytes20(Data memory data) internal pure shift(data, 20) returns(bytes20 value) {
```

Comment:
I wonder where did `shift(data, 20)` come from? This looks suspicious. @abacabadabacaba WDYT?